src/signals/IncidentMap/components/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_ZOOM = 14
```

Comment:
Is min zoom not needed here? Maybe more consistent?

Reply:
No, the min_zoom you referring to is just for the mobile experience (12). The actual min zoom for the map is 8.